recycle-node.rb

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env ruby
+
+
+# Usage:
+#   export AWS_PROFILE=moj-cp
+#   Edit "K8S_CLUSTER_NAME" to specify the cluster to recycle old node. 
+#   Set kubectl config to use-context #K8S_CLUSTER_NAME
```

Comment:
The script can execute the `kubectl config` command, to ensure the context matches the cluster name. This is safer than requiring people to read the comments.

Also,  please remove the reference to `create-cluster.rb`